script/getVersions.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+const semver = require('semver')
+const superagent = require('superagent')
+
+const url = 'https://api.github.com/repos/nodejs/node/releases'
+
+const fetchReleases = async () => {
+  try {
+    const res = await superagent.get(url).set({
+      'User-Agent': 'superagent'
+    })
+    return res.body
+  } catch (err) {
+    console.error(err)
+  }
+}
+
+module.exports = async (supportedVersions) => {
+  const releases = await fetchReleases()
+  if (!releases) {
+    throw new Error('Something went wrong with relases fetching')
+  }
+
+  return supportedVersions.reduce((prev, el) => {
```

Comment:
I'm not sure this is going to return the latest version, or just any that satisfies the SemVer major

Reply:
It's return first match because `Array.find()` that satisfies the semver major version. In our data there is guarantee that latest version will be first match.